### PR TITLE
Add missing step under adding .onmicrosoft domain

### PR DESCRIPTION
**Pull request by ashrayvk**
Entered a missing step under adding a .onmicrosoft.com domain

### CODE REVIEW

File: microsoft-365/admin/setup/add-domain.md [hunk 1]
```diff
@@ -156,7 +156,9 @@ To add an onmicrosoft.com domain:
 
 1. Go to the Microsoft admin center, **Settings** > <a href="https://go.microsoft.com/fwlink/p/?linkid=834818" target="_blank">**Domains**</a>.
 
-2. On the **Overview** tab, select **Add onmicrosoft.com domain**.
+2. Click on an existing *.onmicrosoft.com* domain.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/add-domain.md]:
```suggestion
2. Select an existing *.onmicrosoft.com* domain.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ashrayvk, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@ashrayvk Gentle reminder

**Comment by yogkumgit** [comment]
@tiburd Please help. Thanks

File: microsoft-365/admin/setup/add-domain.md [hunk 2]
```diff
@@ -156,20 +156,20 @@ To add an onmicrosoft.com domain:
 
 1. In the Microsoft 365 admin center, select **Settings**, and then select **Domains**.
 
-2. On the **Domains** page, select your current ***.onmicrosoft.com (Default)** domain.
+1. Click on an existing *.onmicrosoft.com* domain.
```
Comment (tiburd) [inline microsoft-365/admin/setup/add-domain.md]:
```suggestion
1. Select an existing *.onmicrosoft.com* domain.
```

File: microsoft-365/admin/setup/add-domain.md [hunk 3]
```diff
@@ -156,20 +156,20 @@ To add an onmicrosoft.com domain:
...
-2. On the **Domains** page, select your current ***.onmicrosoft.com (Default)** domain.
+1. Select an existing *.onmicrosoft.com* domain.
 
-3. On the **Overview** tab, select **Add onmicrosoft.com domain**.
+1. On the **Overview** tab, select **Add onmicrosoft.com domain**.
 
     ![Domains page.](../../media/onmicrosoft-domains.png)
   
-4. In the **Add onmicrosoft domain** page, in the **Domain name** box, type the name for your new onmicrosoft.com domain. 
+1. In the **Add onmicrosoft domain** page, in the **Domain name** box, type the name for your new onmicrosoft.com domain.
```
Comment (tiburd) [inline microsoft-365/admin/setup/add-domain.md]:
```suggestion
1. On the **Add onmicrosoft domain** page, in the **Domain name** box, enter the name for your new onmicrosoft.com domain. 
```

**Pull request merged**